Fixed #3150 (winding pack current density constraint)

mkovari opened the pull request:
Fixed #3150.  
Removed obsolete references to PROCESS Users's Guide.

jonmaddock commented on process/current_drive.py:
```diff
@@ -1279,8 +1278,7 @@ def iternb(self):
         neutral beam system, based on the 1990 ITER model.
-        AEA FUS 251: A User's Guide to the PROCESS Systems Code
-        ITER Physics Design Guidelines: 1989 [IPDG89], N. A. Uckan et al,
+                ITER Physics Design Guidelines: 1989 [IPDG89], N. A. Uckan et al,
```
Fix indentation please.

jonmaddock commented on process/evaluators.py:
```diff
@@ -37,8 +37,7 @@ def fcnvmc1(self, n, m, xv, ifail):
         constraints in conf.
-        AEA FUS 251: A User's Guide to the PROCESS Systems Code
-        :param n: number of variables
+                :param n: number of variables
```
Again, fix indentation please.

jonmaddock reviewed (changes requested):
I think there's some indentation that needs correcting. Also, why has an image been put in the examples directory? Perhaps this belongs in the docs?

mkovari commented:
The image files plot_proc_1, 2 and 3 must have been put into the examples folder automatically when I ran `pytest`.  Because there were so many modified files I used `git commit -a`, and because I didn't have the examples folder open in VSCode I didn't see they had been changed.  

However, they seem to fit, and since only the new third page is shown in the changes tab, I suppose that the other two pages, plot_proc_1.png and plot_proc_2.png were already there, so I have left it there.

mkovari commented:
I have removed extra indentations from six files.

jonmaddock reviewed (changes requested):
Please fixup the changes to comments to eliminate the 2 correcting commits and eliminate churn. The `plot_procX.png` figures should be created by running the `examples.ipynb` notebook, and the resulting pngs displayed in the notebook. Please look at how figure 1 and 2 are being handled by the notebook to match. Thanks!

mkovari commented:
The plot_proc figures have nothing to do with this pull request.  They result from Chris's edit.  If the example notebook needs changing it should be done there.  I will delete the unwanted png.

mkovari commented:
Unwanted png removed.  Rebased.

mkovari commented:
I have fixed the conflict localy, but when I rebased and then tried to force push, I couldn't do it:
```
(env) mkovari@J1714:~/PROCESS$ git push --force
ssh: connect to host github.com port 22: Connection timed out
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

timothy-nunn commented on source/fortran/init_module.f90:
```diff
@@ -111,7 +111,7 @@ subroutine init
 
 
-      !! AEA FUS 251: A User's Guide to the PROCESS Systems Codefile:///home/mkumar/process/source/fortran/divertor_ode.f90
+      !! file:///home/mkumar/process/source/fortran/divertor_ode.f90
```
Can we just remove this line, that file is long gone
